client/js/gdpr/chart.js

```diff
@@ -117,8 +118,29 @@ export class GdprChart extends HTMLElement {
     this.clearHover();
   }
 
-  onScroll() {
-    this.clearHover();
+  onMouseWheel(ev) {
+    if (ev.ctrlKey) {
+      const {min, max, pow} = Math;
+      const scale = pow(10, ev.deltaY / 100);
+
+      const finalZoom = min(max(this.zoom * scale, 0.1), 3);
```

Comment:
You should add a brief comment explaining why this calculation is necessary

Reply:
+1, you've got several magic numbers and exponents that could use explaining.